coffee-mode.el

```diff
@@ -1208,6 +1245,10 @@ comments such as the following:
   ;; code for syntax highlighting
   (setq font-lock-defaults '((coffee-font-lock-keywords)))
 
+  ;; fix comment filling function
+  (set (make-local-variable 'comment-line-break-function)
+        #'coffee-comment-line-break-fn)
+  (set (make-local-variable 'auto-fill-function) #'coffee-auto-fill-fn)
```

Comment:
auto-fill-mode is enabled automatically if `auto-fill-function` is non-nil. Someone does not expect this behaviour like #341. I have changed from `auto-fill-function` to `normal-auto-fill-function` at #342.  If you want to use auto-fill-mode, please add following configuration.

```lisp
(add-hook 'coffee-mode-hook 'auto-fill-mode)
```